### ISSUE TITLE
Post comments instead of request changes

### ISSUE THREAD

**Issue by justinchuby** [parkerbxyz/suggest-changes#36]
In this example PR https://github.com/onnx/onnx/pull/6377, the bot left a comment to request change, but there was no resolution of the request change status after the comments were addressed. It would be great if there is an option for the bot to post a "comment" instead of "request for change" response, or is that possible already?

**Comment by parkerbxyz** [comment]
The blocking review from the bot can be [dismissed](https://docs.github.com/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/dismissing-a-pull-request-review).

![Image](https://github.com/user-attachments/assets/5f90a7da-b07b-45ef-b4bf-97739b66dbfe)

That being said, I think it makes sense to make the review more configurable so it can be non-blocking. Pull requests welcome!

**Comment by parkerbxyz** [comment]
I've released [v2.0.0](https://github.com/parkerbxyz/suggest-changes/releases/tag/v2.0.0), which changes the default pull request review action from `REQUEST_CHANGES` to `COMMENT`. You can use the `REQUEST_CHANGES` or `APPROVE` review actions by supplying those value to the new the `event` input, like so:

```yaml 
- uses: parkerbxyz/suggest-changes@v2
  with:
    event: 'REQUEST_CHANGES'
```
